docs/build-deploy-pipeline.md

```diff
@@ -6,6 +6,12 @@
 * Any successful build of `master` branch get deployed to Staging
 * Any successful build of a Tag matching a valid SemVer pattern (`vX.Y.Z`) (case-sensitive) gets deployed to Production
 
+## Daily build
+
+There is a cron job set up on Travis that is going to generate a production build from the latest release.
```

Comment:
Can we document _where_ that cron job is and how to modify it if we ever needed to? If it's something we'd rather not make public, we could potentially do so somewhere in Buildit's internal Confluence wiki. However, for future maintenance, I think it's important that this stuff is written down somewhere.

Reply:
Yeah, specifying that it is accessible here https://travis-ci.org/buildit/buildit/settings is probably enough.  Probably not much more to document than that.